docs/source/markdown/podman-machine-inspect.1.md

```diff
@@ -25,10 +25,8 @@ Print results with a Go template.
 
 | **Placeholder**     | **Description**                                                       |
 | ------------------- | --------------------------------------------------------------------- |
```

Comment:
the changes here should probably make a release note

Reply:
Fixed.